examples/swapi/swapi-server.js

```diff
@@ -20,9 +20,16 @@ const createSWAPIServer = () => {
             director: String
         }
 
+        type FilmV2 {
+            title: String
+            episodeNumber: Int
+            director: String
+        }
```

Comment:
might be worth a comment to explain what's going on here / why v2 exists

Reply:
deleted `type FilmV2` as it's a duplicate of Film